data/core/units.cfg

```diff
@@ -116,6 +117,29 @@ A few elves, venturing far down the paths of magic and mysticism, become sensiti
 
 Elves spend much of their time honing their talents and skills. Those not adept at the magical arts typically devote their time honing their physical skills. As a result, elves excel at archery, which is perhaps their most important method of warfare. Most elvish troops carry a bow and no other race can rival their archers in speed and accuracy. All elves also share an intense affection for unspoiled nature. They often feel uncomfortable in open, unvegetated spaces. They live primarily in the forests of the Great Continent; the Aethenwood in the southwest, Wesmere in the northwest, and the great northern woods of which the Lintanir Forest is the southernmost edge.
 
+Elves are the eldest race of the continent, with the possible exception of trolls. Many of their settlements cannot be reliably dated, undoubtedly having existed for over a millennium."
+        num_traits=2
+        markov_chain_size=2
+        {ELVISH_NAMES}
+        {TRAIT_DEXTROUS}
+    [/race]
+
+    # wmllint: general spellings elf elves elvish
+    # wmllint: general spellings Kaleh Nym Garak Zhul
+    [race]
+        id=quenoth
+        male_name= _ "race^Quenoth Elf"
+        female_name= _ "race+female^Quenoth Elf"
+        plural_name= _ "race^Quenoth Elves"
+        # wmllint: local spelling unvegetated
+        description= _"Compared to humans, elves are somewhat taller, more agile but less sturdy. They have slightly pointy ears, pale skin and usually blond hair. Few differences between humans and elves are more pronounced than the Elves’ unusually long life — most, unless claimed by illness, accident or war, live a full two and a half centuries. While some elves possessing a high magical aptitude have been known to live an additional full century, most elves begin to grow physically frail at some point between 250 and 300 years of age and pass away rapidly (generally within a year or two) thereafter.
```

Comment:
That's a placeholder – they will need their own description.

Reply:
Yes. Nemaara agreed to write the description.